workspace/apps/pidp/src/app/features/organization-info/pages/endorsements/endorsements.page.scss

```diff
@@ -100,41 +110,152 @@ p {
       }
       & .tab-content {
         padding: var(--gap);
-        display: flex;
-        flex-wrap: wrap;
         gap: var(--gap);
-        flex-direction: var(--tab-content-flex-direction);
-
-        & .endorsement-card {
-          flex-basis: calc((100% - (var(--gap))) / 2);
-
-          @include media-breakpoint-up(lg) {
-            flex-basis: calc((100% - (var(--gap))) / 3);
-          }
-
-          display: flex;
-        }
+        display: grid;
+        grid-template-columns: calc(50% - 15px) calc(50% - 15px);
       }
     }
     & .endorsement-request {
       & header {
         margin: calc(2 * var(--gap)) 0 var(--gap);
       }
-      & p,
-      & h3 {
-        padding: var(--endorsement-request-item-padding);
-      }
       & mat-form-field {
-        padding: var(--endorsement-request-item-padding);
-        width: var(--endorsement-form-field-width);
+        max-width: var(--endorsement-form-field-width);
+        width: 100%;
       }
     }
     & .actions {
-      display: flex;
-      padding: var(--endorsement-request-item-padding);
+      margin-top: 2rem;
+      button {
+        border-radius: 0.25rem;
+        border: 2px solid #036;
+        font-size: 1rem;
+        font-weight: 700;
+        letter-spacing: 0.0625rem;
+      }
       & button:not(:first-of-type) {
         margin-left: var(--gap);
       }
     }
   }
 }
+.viewport-large {
+  --tab-content-flex-direction: row;
+  & .content {
+    & .care-team {
+      & .care-team-tabs {
+        ::ng-deep .mat-mdc-tab-group {
```

Comment:
So ng-deep has been [deprecated](https://v17.angular.io/guide/component-styles#deprecated-deep--and-ng-deep) and is not recommended for use even if it still works for some css rules, because it completely disables view-encapsulation for that rule. For css rules without encapsulation we store them in this file `_material-overrides.scss`, as it could get cumbersome to track down a global rule overriding your style, if isn't placed in a file with global view. 

Generally speaking, try applying a class to the element you want to apply style to, target it to increase specificity and if the style is style not applying, then can add the global style for it.